contracts/mocks/AddressImpl.sol

```diff
@@ -13,6 +13,10 @@ contract AddressImpl {
         Address.sendValue(receiver, amount);
     }
 
+    function functionCall(address target, bytes calldata data) external returns (bytes memory) {
+        return Address.functionCall(target, data);
```

Comment:
Because this function is not `view`, accessing the returned data is not possible. The easiest way to test this is to emit an event including it. See for example the tests for `EnumerableSet`.

Reply:
@julianmrodri this is what you were asking about

Reply:
Thanks @nventuro will look into this. The rest of the tests are kind of OK-ish?